ratpack-core/src/main/java/ratpack/http/client/RequestSpec.java

```diff
@@ -117,6 +117,12 @@ default RequestSpec sslContext(Factory<SSLContext> factory) throws Exception {
 
   URI getUrl();
 
+  default RequestSpec connectTimeoutSeconds(int seconds) {
```

Comment:
AFAICT, if this isn’t set it falls back to the socket’s SO_TIMEOUT value, whose default is different on different machines.